src/usr/share/gpgmailer/gpgmailer-daemon.py

```diff
@@ -416,11 +418,11 @@ def check_if_mounted_as_tmpfs(pathname):
     return 'none on {0} type tmpfs'.format(pathname) in subprocess.check_output('mount')
 
 
-# TODO: Add a switch to not spool to a RAM disk. (issue 21)
-def create_spool_directories(program_uid, program_gid):
+def create_spool_directories(config, program_uid, program_gid):
     """Mounts the program spool directory as a ramdisk and creates the partial and outbox
     subfolders. Exit if any part of this method fails.
 
+    config: The program configuration dictionary to read the use_tmpfs_spool flag from.
```

Comment:
Is it necessary to pass the entire config dictionary for one boolean? I think simply passing the one value we need is clearer here.